### ISSUE TITLE
Adjust/append the 'user level' information in details accordingly

### ISSUE THREAD

**Issue by tooomm** [Cockatrice/Cockatrice#2342]
If you right click somebody, the `User details`  looks like this for a donator:
![user_infor](https://cloud.githubusercontent.com/assets/9874850/21647574/29aece94-d29b-11e6-91fb-070d92cf5968.png)

**It correctly displays for Admin and Moderator, but not for the new ranks.**

Since you needs to be registered for all ranks, I suggest to think about something like this:
Administrator / VIP (used to be Administrator)
Administrator
Moderator / VIP (used to be Moderator)
Moderator
Donator (used to be Registered user)
VIP (used to be Registered user)
Normal (used to be Registered user)
Unregistered

**Comment by skwerlman** [comment]
AFAIK, this is because donator/VIP aren't ranks in the same sense as admin or mod. they instead are more like "rank modifiers".

I agree that donator/VIP should be displayed there instead of registered user.

**Comment by woogerboy21** [comment]
> AFAIK, this is because donator/VIP aren't ranks in the same sense as admin or mod. they instead are more like "rank modifiers".

This is correct.  The user levels are still unreg/reg/mod/admin .  Maybe user level in the UI is slightly misleading at this point with the introduction of the new code functionality.

**Comment by tooomm** [comment]
If it's even not that clear to people close to the project - and I hear some uncertainty in recent times - how can it be understood by the ordinary user, mods or other server admins (e.g. for smaller playgroups or even other public servers, whose owners doesn't happen to hang around here all the time)?

Maybe we should clear up things a bit for all stakeholders...

**Comment by woogerboy21** [comment]
@tooomm 
Im not sure how its unclear.  It so far seems clear to me and @skwerlman .  It sounds to me like your just missing the understanding or trying to accomplish some goal im not understanding.  You wanted some indicator that the user is of a different user level (its there as registered user) and you would like to understand the users priv level (its there as the gummy icon).

The app has a rank (your terms not mine) of unregistered user, registered user, moderator, and admin.  And with in each of those there is a sub tier.  You have NONE, VIP, or DONATOR.  Not sure how unclear that is?

**Comment by woogerboy21** [comment]
So you have something like this:

Administrator
  - VIP
  - DONATOR
  - NONE
Moderator
  - VIP
  - DONATOR
  - NONE

Registered User
  - VIP
  - DONATOR
  - NONE

Unregistered

VIP/DONATOR listed above can be interchanged as one listed before the other have no weight in regards to one being better than the other.

**Comment by woogerboy21** [comment]
Oh one other thing to note.  The database has columns in it to store when a user was given the privlevel and when the date/time of the privlevel should expire.  Code wise the server does nothing with this information, it only knows about it.  It's up to the server operator to implement external functionality for doing something with the information.  I included a script in the source files that quires the end time and changes the users privlevel to NONE (which needs updated already btw).  So its up to the server operator on if they wish to have privlevel recognition be removed at any point in time.

**Comment by tooomm** [comment]
Did I say that I don't understand it? ;)
I totally do, but it was not self explanatory and I didn't looked through it in the first run, that's for sure.
You somehow get me wrong though, I'm not mad at you or your feature at all. I just feel that things aren't as easy as they should be.
If I just recap gitter and github it's not perfectly clear to everybody - which I totally understand.

---

Back to topic:
Perfect, so why we don't tell the user exactly that?

```
User level: Unregistered
or
User level: Registered (VIP)
or
User level: Moderator (Donator)
or
User level: Administrator (VIP)
```

*btw that's exactly what I suggested in my starting post... "admin level / privlevel" ...*

Well, at least for admin and mod. I mixed some other thoughts in there as well. But as long as we have a common ground here everything is alright and an improvement to the initial issue. 👍

**Comment by woogerboy21** [comment]
Oh no, dont think I was upset in any way.  I just thought it was clear the way it is presented.  We have indicators for both the userlevel and privlevel either via text and gummy combinations.  If you wanted the text to be updated to indicate privlevel as well should we change the descriptor beside the text?  Since userlevel is only unreg,reg,mod,admin and not none,vip,donator ?

Is the word "rank" a more accurate descriptor?  Thats what I am trying to understand.

**Comment by tooomm** [comment]
>If you wanted the text to be updated to indicate privlevel as well should we change the descriptor beside the text?

It would be more clear to have the text updated as well, yes.

Change the descriptor?
I think I picked the term "rank" up around here or in gitter. But level or rank is pretty much interchangeable here imo.
The sub types are a bit more difficult to group or find an easy to understand term for them.
As long as they are just appended in brackets for example it would be ok as is.
If you prefer a new line with that information I don't know any good term at the moment.

**Comment by tooomm** [comment]
Done with #2349 

@poixen github didn't catch `Fix for #2342`, try `fixes #x` instead!
